InputAnswer/InputAnswer/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace InputAnswer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int excitement = 0;
+             
+
+            Console.WriteLine("Hi! We are starting out on a great journey");
+            Console.WriteLine("You and I");
+            Console.WriteLine("Are you excited?");
+            Console.WriteLine("Type 1 for yes, 2 for no, and 3 for eh: ");
```

Comment:
Info: You can use a thing called a StringBuilder which will be a bit cleaner than multiple Console.WriteLines. 

StringBuilders are useful for stuff like this, though they do need to be "disposed" of, which might be more advanced than we want to get right now. This is just info for you to look into.